src/components/CompareResults/beta/RevisionSelect.tsx

```diff
@@ -25,31 +25,66 @@ const styles = {
   }),
 };
 
+interface RevisionOption {
+  [rev: string]: {
+    key: string;
+    text: string;
+  };
+}
+
 const allRevisionsOption =
   Strings.components.comparisonRevisionDropdown.allRevisions;
 
-const fakeRevisionsOptions = [
-  allRevisionsOption,
-  'bb6a5e451dace3b9c7be42d24c9272738d73e6db',
-  '9d50665254899d8431813bdc04178e6006ce6d59',
-  'a998c42399a8fcea623690bf65bef49de20535b4',
-];
```

Comment:
Maybe you could keep it as:
```js
const fakeRevisionsOptions = [
  'bb6a5e451dace3b9c7be42d24c9272738d73e6db',
  '9d50665254899d8431813bdc04178e6006ce6d59',
  'a998c42399a8fcea623690bf65bef49de20535b4',
];
```
and pass it to `formatRevisionsOptions` later:
```js
  const revisionsOption = formatRevisionsOptions(fakeDataParam ? fakeRevisionsOptions : newRevisions);
```
(of course you need to move up the getter for the fake data param, and also remove line 81).

What do you think?